seametrics/tracking/utils.py

```
@@ -301,22 +307,81 @@ def compute_metrics_by_sequence(
+    }
+
+    for sequence_name in sequence_list:
+        sequence_view = view.match(F("sequence") == sequence_name)
+        for pred_field in pred_fields:
+            gt, pred = compute_metrics(
+                view=sequence_view, gt_field=gt_field, pred_field=pred_field
+            )
+            for instance in instances[pred_field].values():
+                try:
+                    instance.update(gt, pred, sequence_name)
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)